### DISCUSSION TITLE
Perfect course about PyTorch!

### DISCUSSION THREAD

**Discussion by Taemin90** [mrdbourke/pytorch-deep-learning#184] [General]
Thank you for your lecture!
It was perfect and very helpful for me.

Are you planning Chapter 05~09 lectures?
I hope they will come up :)

Thank you again

**Comment by satvikpendem** [comment]
He already has those lectures, they are paid though: https://www.udemy.com/course/pytorch-for-deep-learning/

**Comment by mrdbourke** [comment]
Hey @Taemin90,

Thank you for the kind words!

Lectures for sections 00-04 are on YouTube.

And @satvikpendem is right, lectures for sections 05-09 are available as a paid upgrade.

You can get them on: 

* Zero to Mastery - https://dbourke.link/ZTMPyTorch 

Or 

* Udemy - https://www.udemy.com/course/pytorch-for-deep-learning/
